### PR TITLE
Fix new feedforwardgroup options

### PR DESCRIPTION
**Pull request by Asizon**
I dont know were and why we loose msp write lines for these two new feedforward options.
Maybe we loose afeter this @haslinghuis  ??? is really weird. Also added these 2 new parameters to fc.js
https://github.com/betaflight/betaflight-configurator/commit/92948e22129cd4d0081bb5ce278536bba9dd5aa2

### CODE REVIEW

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Thanks, Asizon.  
Tested and checked, all feedforward values save correctly. 
I also checked all the othervalues in the PID_tuning page, they seem to save and refresh correctly.

**Comment by limonspb** [comment]
i think Asizon is correct, they disappeared in this PR in the second commit:
https://github.com/betaflight/betaflight-configurator/pull/2638
@haslinghuis  could you please check if something else was unintentionally removed in that merge commit.

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
I guess rebase went wrong (thinking it wasn't merged yet), As this already was introduced here:
https://github.com/betaflight/betaflight-configurator/pull/2656

**Pull request merged**
